docs/guides/how_to_guides/creating_and_editing_expectations/how_to_edit_an_expectation_suite_using_a_disposable_notebook.rst

```diff
@@ -15,53 +15,160 @@ To simplify this workflow, the CLI command ``suite edit`` takes a named :ref:`Ex
+
+            For convenience, the :ref:`Data Docs <reference__core_concepts__datasources>` page for each Expectation Suite has the CLI command syntax for you. Simply press the “How to Edit This Suite” button, and copy/paste the CLI command into your terminal.
+
+        2. Run the boilerplate code at the top of the notebook.
+
+            The first code cell at the top of the notebook contains boilerplate code to fetch your Expectation Suite for you to edit in the subsequent cells.
+
+            Run this cell to get started.
+
+        3. Run (and edit) any Expectations in the "Create & Edit Expectations" section:
+
+            The middle portion of the notebook contains a code cell to reproduce each Expectation in the Suite.
+
+            - If you re-run these cells with no changes, you will simply recreate all of your existing Expectations (without validations, since there is no Batch of data). At the end of your notebook, the Expectation Suite will contain exactly the same Expectations that you started out with.
+            - If you edit and run any of the Expectations, the new parameters will replace the old ones.
+            - If you don't run an Expectation, it will be left out of the Suite.
+            - You are welcome to modify the existing Expectations and/or add new ones using the `ExpectationConfiguration` syntax (see :ref:`How to create a new Expectation Suite without a sample Batch <how_to_guides__creating_and_editing_expectations__how_to_create_a_new_expectation_suite_without_a_sample_batch>`).
+
+            Until you run the final cell in the notebook, any changes will only affect the Expectation Suite in memory. They won't yet be saved for later.
+
+        4. Save your Expectations and review them in Data Docs
+
+            At the bottom of the notebook, there's a cell labeled "Save & Review Your Expectations." It will:
+
+                #. Save your Expectations back to the configured Expectation Store.
+                #. Rebuild Data Docs.
+                #. Open your Data Docs to the ExpectationSuite page for your review.  (Note: since there is no Batch of data in this mode, validations are not run. Nevertheless, when the Batch of data is available, all your expectations will appear intact, with the ability to run validations in the notebook.)
+
+        5. Delete the notebook
+
+            In general, these Jupyter notebooks should not be kept in source control. In almost all cases, it's better to treat the Expectations as the source of truth, and delete the notebook to avoid confusion. (You can always auto-generate another one later.)
+
+            The notebook will be stored in the ``great_expectations/uncommitted`` directory. You can remove it like so:
+
+            .. code-block:: bash
+
+                rm great_expectations/uncommitted/edit_movieratings.ratings.ipynb
+
+    .. tab-container:: tab2
+        :title: Show Docs for V3 (Batch Request) API With a Batch of data
+
```

Comment:
It might be nice to have a corresponding sentence to the one in the prior tab:
> This mode of Great Expectations allows you to edit your Expectation Suite even when the Batch of data is unavailable (or not needed) at the time.

Reply:
@Aylr While it is possible to do so, it is not as easy as changing the command-line flag... So I am not sure how that could be done and feel that it would be advantageous to recommend that the users upgrade to V3...  Please let me know what you think.  Thank you.

Reply:
Ah let me clarify. I wasn't clear enough. I don't mean add this to v2. I mean describe the interactive command being with feedback from data.

Reply:
Addressed!  Thanks!